src/Esfa.Vacancy.Api.Types/Vacancy.cs

```diff
@@ -88,8 +88,8 @@ public class Vacancy
         public int? LarsStandardId { get; set; }
```

Comment:
Rather than LarsStandardId we can call this StandardCode, I used LarsStandardId in the SQL query just so we can distinguish between the internal StandardId and the one that corresponds to LARS.